content/docs/architecture/resources/course.md

```diff
@@ -1,10 +1,35 @@
 +++
 title = "Course"
+description = "A collection of HobbyFarm Scenarios meant to be consumed as a single unit or as a series of individual sessions."
 +++
 
-A course is a collection of scenarios meant to be consumed as a single unit or as a series of individual sessions (configurable). A course is useful in situations where you want to group related learnings together or provide a day's worth of curriculum to students who can retain their VMs from scenario to scenario. 
+A `Course` is useful in situations where a trainer wants to group related learnings together or provide a day's worth of curriculum to students who can retain their VMs from scenario to scenario.
 
-Here's an example course:
+The Course resource is used to populate content in the HobbyFarm Admin-UI Content Management > Courses page. Modifying a Course resource in Kubernetes will modify the information in the Admin-UI.
+
+## Kubernetes Commands
```

Comment:
these commands are the same for every CRD. can we move it to resources/_index or into their own section? it seems pretty duplicate

also the namespace depends on the helm installation. might by `hobbyfarm-system` might be something different

Reply:
@jggoebel I agree that it's redundant. My thought process was an assumption that someone might be referencing a page independently while researching a specific topic, and may not have reviewed any other pages up to this point-in-time.

I'm also making an assumption that someone may be attempting to deploy HobbyFarm who is not familiar with Kubernetes as a whole, which is a distinct possibility based on my experience training both private and public sector engineers.

Reply:
I have a note in the deployment documentation regarding the use of the `namespace`. It states the following:

```
The namespace name is arbitrary and can be changed to suit your needs. However, be sure to use the same namespace name throughout the installation process.
```

Should I clarify this anywhere else that the namespace example is referenced, or is this example good enough?  I could also flush out the documentation note further, explaining that HobbyFarm can be deployed in multiple namespaces. Would this provide any added benefit?

Reply:
Closing this as complete per our discussion in the call today.